src/recipes/banana-oatmeal-cookies.md

```diff
@@ -0,0 +1,53 @@
+
+---
+tags: ["2 Ingredient", "cookies"]
```

Comment:
hi, can you categorize this just as 'Dessert' please, to keep the tags simple :)

Reply:
Dessert instead of cookies? Still include '2 Ingredient' ?